src/main/java/mat/server/service/impl/MeasurePackageServiceImpl.java

```diff
@@ -556,8 +556,10 @@ private void createAndSaveExportsAndArtifacts(MeasureExport export, boolean shou
         measure.setExportedDate(new Date());
         measureDAO.save(measure);
         measureExportDAO.save(export);
-        //New for FHIR we create a CQL_LIBRARY in draft state for the measure so microservices can find it.
-        measureLibraryService.exportDraftCQLLibraryForMeasure(measure);
+        if (measure.isFhirMeasure()) {
```

Comment:
@cday-semanticbits 
JFYI - this custom FHIR code caused issues to QDM packaging so we now run it for FHIR measured only

Reply:
Good catch, should have been for FHIR only.